### PR TITLE
Improve coordination between java-versions and idea ipr

### PR DESCRIPTION
**Pull request by carterkozak**
## Before this PR
ipr imports failed to build due to skewed bytecode versions, requiring manual edits to ipr files and UI costomization.

## After this PR
"it just works"
==COMMIT_MSG==
Improve coordination between java-versions and idea ipr allowing project iprs to be successfully imported
==COMMIT_MSG==

## Possible downsides?
I don't actually know groovy, and write java with a different extension. I suspect there's some clever syntax that makes this easier ¯\_(ツ)_/¯

### CODE REVIEW

**Comment by ferozco** [comment]
👍

**Comment by ferozco** [comment]
Awesome thanks @carterkozak

File: gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy [hunk 1]
```diff
@@ -18,9 +18,17 @@ package com.palantir.baseline.plugins
...
+import com.palantir.baseline.extensions.BaselineJavaVersionsExtension
 import com.palantir.baseline.util.GitUtils
 import groovy.transform.CompileStatic
 import groovy.xml.XmlUtil
+import org.gradle.api.JavaVersion
+import org.gradle.jvm.toolchain.JavaLanguageVersion
+import org.gradle.plugins.ide.idea.model.IdeaLanguageLevel
+import org.gradle.plugins.ide.idea.model.IdeaModule
+import org.gradle.plugins.ide.idea.model.Jdk
+
```
Comment (tpetracca) [inline gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy:31]: what happened with the imports here?

Comment (fawind) [reply gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy:31]: We don't have a formatter for groovy that automatically cleans up imports.

**Pull request merged**